.circleci/config.yml

```diff
@@ -4,14 +4,14 @@ executors:
     environment:
       IMAGE_NAME: suldlss/dor-indexing-app
     docker:
-    - image: circleci/buildpack-deps:stretch
+    - image: cimg/base
```

Comment:
not sure this is going to work when this PR is merged to main and a docker image needs to be created;  see https://github.com/sul-dlss/dlme/pull/1382 and maybe talk to @aaron-collier